apps/javascript/src/challenges/dynamic-tooltip/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
https://github.com/sadanandpai/frontend-mini-challenges/blob/main/CONTRIBUTING.md#steps-to-contribute-javascript-challenge

Please go through the instructions